amqp/subscriber.go

```diff
@@ -28,7 +30,7 @@ type Subscriber interface {
 	SubscribeDeviceUplink(appID, devID string, handler UplinkHandler) error
 	SubscribeAppUplink(appID string, handler UplinkHandler) error
 	SubscribeUplink(handler UplinkHandler) error
-	ConsumeUplink(queue string, handler UplinkHandler) error
+	ConsumeMessages(queue string, uplinkHandler UplinkHandler, eventHandler DeviceEventHandler) error
```

Comment:
It is not easier to implement both are straightforward.
It harder to manage since it will require more configuration and management, for example the integration controller will have to manage two queue and the process to subscribe to both.
The difference is minimal in term of complexity, but if it aimed for performance then it's more efficient to use two queues and depend on the broker.